docs/src/user-docs/guides-using-presto.md

```
@@ -179,6 +200,21 @@ contain the field `foo.bar`, you can query it using:
 SELECT foo.bar FROM default LIMIT 1;
 ```
 
+### Querying from the UI
+The CLP UI is available at [http://localhost:4000](http://localhost:4000) (if you changed
+`webui.host` or `webui.port` in `etc/clp-config.yml`, use the new values).
+
+### Querying from the Presto CLI
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)